examples/all/proximity_sensor_example.cc

```diff
@@ -85,6 +85,7 @@ static void setup_proximity_sensor(Mmio<OpenTitanI2c> i2c, const uint8_t Addr)
 	if (!i2c->blocking_write(ApdS9960I2cAddress, buf, 2, true))
 	{
 		Debug::log("Failed to write proximity sensor address");
+		return;
```

Comment:
Redundant?

Reply:
Yes, but it's good to have in case code gets added below.

Reply:
Ha